<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawOcclusionWidget</class>
 <widget class="QWidget" name="DrawOcclusionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>170</width>
    <height>124</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>170</width>
    <height>124</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>124</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="cbFrustumCulling">
     <property name="text">
      <string>Frustum Culling</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbOcclusionQueries">
     <property name="text">
      <string>Occlusion Queries</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbRealGeometry">
     <property name="text">
      <string>Query real geometry</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbDrawInformation">
     <property name="text">
      <string>Draw information</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
